tests/unit/test_hyper_transformer.py

```diff
@@ -2301,7 +2301,7 @@ def test_remove_column_in_multi_column_fields_single_column_left(self):
         assert ht._multi_column_fields == expected_column_in_tuple
 
     def test_update_transformers_by_sdtype_with_multi_column_transformer(self):
-        """Test ``update_transformers_by_sdtype`` with columns use with a multi-column transformer."""
+        """Test ``update_transformers_by_sdtype`` with columns use with a multicolumn transformer"""
```

Comment:
what changed this?

Reply:
We started checking docstring length and this line was too long. I shorted it.

Reply:
Ok, can you revert for consistency? If you need to shorten, you can get rid of the ``update_transformers_by_sdtype`` part since it's redundant